manifest.md

```diff
@@ -46,18 +46,22 @@ Unlike the [image index](image-index.md), which contains information about a set
     While an empty blob (`size` of 0) may be preferable, practice has shown that not to be ubiquitiously supported.
     Instead, the blob payload can be the most minimal content that is still valid JSON object: `{}` (`size` of 2).
     The blob digest of `{}` is `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`.
-    See the [example SCRATCH config](#example-of-a-scratch-config-or-layer-descriptor) below, and the `ScratchDescriptor()` of the reference code.
+    See the [example SCRATCH config](#example-of-a-scratch-config-or-layer-descriptor) below, and `ScratchDigestSHA256`/`ScratchData` of the reference code.
 
 - **`layers`** *array of objects*
 
     Each item in the array MUST be a [descriptor](descriptor.md).
-    The array MUST have the base layer at index 0.
-    Subsequent layers MUST then follow in stack order (i.e. from `layers[0]` to `layers[len(layers)-1]`).
-    The final filesystem layout MUST match the result of [applying](layer.md#applying-changesets) the layers to an empty directory.
-    The [ownership, mode, and other attributes](layer.md#file-attributes) of the initial empty directory are unspecified.
+    The array MUST have at least one entry.
```

Comment:
I think this needs to be a SHOULD or even left ambiguous. Honestly in the long term I would like us to consider removing this and make a 2.0 with this as optional array since originally that might have been the interpretation. 
There are implementations that was shared during the call 
Quoting @tianon from zoom call chat :) - "I know I can make a zero-layer image with just `docker build` (or `docker buildx build`), so there's already both tooling and registry precedence for those working 😅"

Honestly if we were to change expectations of clients or authors reading the specification (since its currently ambiguous), I would personally move towards OPTIONAL given #1025.